modules/core/test/v2/unit/baseCoin.ts

```diff
@@ -220,4 +220,12 @@ describe('V2 Base Coin:', function () {
     });
   });
 
+  describe('getBaseAddress', function () {
```

Comment:
please move to `stx` test file since we shouldn't have this function on `baseCoin`